zerver/lib/push_notifications.py

```diff
@@ -659,12 +659,12 @@ def get_gcm_alert(
     display_recipient = get_display_recipient(message.recipient)
     if (
         message.recipient.type == Recipient.HUDDLE
-        and trigger == NotificationTriggers.PRIVATE_MESSAGE
+        and trigger == NotificationTriggers.DIRECT_MESSAGE
     ):
         return f"New direct group message from {sender_str}"
     elif (
         message.recipient.type == Recipient.PERSONAL
-        and trigger == NotificationTriggers.PRIVATE_MESSAGE
+        and trigger == NotificationTriggers.DIRECT_MESSAGE
```

Comment:
Hmm, where is the compatibility code for handling "private_message" in a push notifications queue?

Reply:
Hm, the compatibility code was missing.

Added the compatibility code + did manual testing by changing the trigger value in `get_push_notification_trigger`